go-controller/pkg/ovn/namespace.go

```diff
@@ -244,12 +244,5 @@ func (oc *Controller) getNamespaceLock(ns string) *sync.Mutex {
 
 	// lock the individual namespace
 	mutex.Lock()
-
```

Comment:
Should we change `deleteNamespace` to use `getNamespaceLock` as well?

Reply:
@abhat I thought about that, but it needs to hold the namespaceMutexMutex across the delete of the namespace lock from the map, so it ended up not being very workable.

Reply:
That makes sense 👍 .